extensions/amp-image-lightbox/0.1/amp-image-lightbox.js

```diff
@@ -869,8 +873,9 @@ class AmpImageLightbox extends AMP.BaseElement {
     this.buildLightbox_();
 
     const source = invocation.caller;
+    const tagName = source.tagName.toLowerCase();
     userAssert(
-      source && SUPPORTED_ELEMENTS_[source.tagName.toLowerCase()],
+      source && (SUPPORTED_AMP_ELEMENTS_.has(tagName) || 'img' === tagName),
```

Comment:
nit: This allowlist is not used anywhere else, so the distinction between AMP/non-AMP elements might not be necessary. Consider adding `img` to the list instead of the special condition.

Reply:
Originally did this, but moved away since it was strange to have a mix of AMP Elements and HTMLElements.

Will change though, this seems like a good thing to address.